Projects/Home/Sources/Claims/ClaimDetailHeader.swift

```diff
@@ -0,0 +1,137 @@
+import SwiftUI
+import hCore
+import hCoreUI
+import hGraphQL
+
+struct ClaimDetailHeader: View {
+    init(
+        title: String,
+        subtitle: String,
+        submitted: String?,
+        closed: String?,
+        payout: MonetaryAmount?
+    ) {
+        self.title = title
+        self.subtitle = subtitle
+        self.submitted = submitted
+        self.closed = closed
+        self.payout = payout
+    }
+
+    let title: String
+    let subtitle: String
+    let submitted: String?
+    let closed: String?
+    let payout: MonetaryAmount?
+
+    private var submittedDate: String {
+        let dateFormatter = DateFormatter.withIso8601Format("yyyy-MM-dd'T'HH:mm:ss.SSSSSSZ")
+        guard let submitted = submitted,
+            let date = dateFormatter.date(from: submitted)
+        else { return "-" }
+        return readableDateString(from: date)
+    }
+
+    private var closedDate: String {
+        let dateFormatter = DateFormatter.withIso8601Format("yyyy-MM-dd'T'HH:mm:ss.SSSSSSZ")
+        guard let closed = closed,
+            let date = dateFormatter.date(from: closed)
+        else { return "-" }
+        return readableDateString(from: date)
+    }
+
+    /// Converts date into a readable friendly string
+    private func readableDateString(from date: Date) -> String {
```

Comment:
I remember that we have some very similar code in hCore, maybe we can reuse that?

Reply:
I needed to use RelativeDateTimeFormatter for this particular problem. I haven't found it to be used else where in the app. Do you know of any particular name by which I can find similar code?

If not, I can put this code in hCore